### PR TITLE
Added references to logging daemonsets

### PR DESCRIPTION
**Pull request by agup006**
Added Fluent-bit, Fluentd Enterprise as well as references to the website links

> NOTE: Please check the “Allow edits from maintainers” box (see image below) to  
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4254)

### CODE REVIEW

**Comment by agup006** [comment]
Signed CLA - thanks!

File: docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ collected.  Deleting a DaemonSet will clean up the pods it created.
 Some typical uses of a DaemonSet are:
 
 - running a cluster storage daemon, such as `glusterd`, `ceph`, on each node.
-- running a logs collection daemon on every node, such as `fluentd` or `logstash`.
+- running a logs collection daemon on every node, such as (`fluentd`)[http://www.fluentd.org], (`fluent-bit`)[http://fluentbit.io/documentation/current/kubernetes/], (`fluentd-enterprise`)[http://fluentd.treasuredata.com] or `logstash`.
```
Comment (chenopis) [inline docs/concepts/workloads/controllers/daemonset.md]: The link syntax is backwards, it should be \[link title](URL), i.e.:

...running a logs collection daemon on every node, such as \[\`fluentd\`](http://www.fluentd.org), \[\`fluent-bit\`](http://fluentbit.io/documentation/current/kubernetes/), \[\`fluentd-enterprise\`](http://fluentd.treasuredata.com) or \`logstash\`.

**Comment by chenopis** [comment]
@agup006 BTW, you can preview the page here: https://deploy-preview-4254--kubernetes-io-master-staging.netlify.com/docs/concepts/workloads/controllers/daemonset/#what-is-a-daemonset

**Comment by crassirostris** [comment]
@agup006 I think this is not the right place to add those links, because it's out of the context of this article.

We're going to work out soon how to present link to the logging solution in the best possible way and place in the one, well advertised to the users, place. Does it make sense?

**Comment by piosz** [comment]
I agree with @crassirostris. This is not the place where multiple logging solutions should be enumerated, because this is not a page about logging. We are happy to accept a contribution to our [logging documentation](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/concepts/cluster-administration/logging.md), though unfortunately it's not in a shape where you can easily add another solution. We plan to rewrite it a bit.

We are happy to cooperate if you are interested. In the meantime I'm closing the PR.

**Pull request closed without merging**